### PR TITLE
Add zap to CIPD setup

### PR DESCRIPTION
**Pull request by tpudlik**
This makes the tool available for running the tests in CI.

@andreilitvin

### CODE REVIEW

File: scripts/zap.json
Comment (bzbarsky-apple) [inline scripts/zap.json:6]: This is going to need to change often.  Please update the zap version update script so it updates this too.

**Review by andy31415 — CHANGES_REQUESTED** [review]
I believe this will be hard to maintain - this would require a fucsia push whenever we want zap updates and unless we get more permissions to do that (i.e. beyond fucsia team) it is probably not something we can manage.

**Review by andy31415 — APPROVED** [review]
Approved: had offline chat about this - CIPD is automated to get the latest release of zap within 4 hours ... this seems to actually speed up our development cycle because we stop requiring a separate docker file.

File: scripts/zap.json
Comment (tpudlik) [reply scripts/zap.json:6]: Done! Thanks for pointing this out.

**Comment by tpudlik** [comment]
@andy31415 I updated the zap update script, but will leave any Dockerfile and documentation updates to you.

**Comment by tpudlik** [comment]
I believe the only blocker is that I need approval from another reviewer.

@bzbarsky-apple could you approve (assuming I've addressed your concerns!)?

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**
